flink-connectors/flink-connector-aws-kinesis-firehose/src/test/java/org/apache/flink/connector/firehose/sink/KinesisFirehoseSinkITCase.java

```diff
@@ -78,42 +76,36 @@
     @Before
     public void setup() throws Exception {
         System.setProperty(SdkSystemSetting.CBOR_ENABLED.property(), "false");
-        httpClient = AWSServicesTestUtils.createHttpClient(mockFirehoseContainer.getEndpoint());
-        s3AsyncClient = createS3Client(mockFirehoseContainer.getEndpoint(), httpClient);
-        firehoseAsyncClient = getFirehoseClient(mockFirehoseContainer.getEndpoint(), httpClient);
-        iamAsyncClient = createIamClient(mockFirehoseContainer.getEndpoint(), httpClient);
+        s3HttpClient = createHttpClient(mockFirehoseContainer.getEndpoint());
+        firehoseHttpClient = createHttpClient(mockFirehoseContainer.getEndpoint());
+        iamHttpClient = createHttpClient(mockFirehoseContainer.getEndpoint());
+        s3AsyncClient = createS3Client(mockFirehoseContainer.getEndpoint(), s3HttpClient);
+        firehoseAsyncClient =
+                getFirehoseClient(mockFirehoseContainer.getEndpoint(), firehoseHttpClient);
+        iamAsyncClient = createIamClient(mockFirehoseContainer.getEndpoint(), iamHttpClient);
+        env = StreamExecutionEnvironment.getExecutionEnvironment();
     }
 
     @After
     public void teardown() {
         System.clearProperty(SdkSystemSetting.CBOR_ENABLED.property());
-        AWSGeneralUtil.closeResources(
-                httpClient, s3AsyncClient, firehoseAsyncClient, iamAsyncClient);
+        s3HttpClient.close();
+        firehoseHttpClient.close();
+        iamHttpClient.close();
+        s3AsyncClient.close();
+        firehoseAsyncClient.close();
+        iamAsyncClient.close();
```

Comment:
Why not use the util to close these?

Reply:
I will do.

Reply:
This was resolved but not updated?

Reply:
Sorry, lack of time is not an excuse for not doing things properly...

Reply:
Since we need to close the additional clients, please make the change